common/viperutil/config_test.go

```diff
@@ -29,30 +31,23 @@ func TestEnvSlice(t *testing.T) {
 		}
 	}
 
-	envVar := "VIPERUTIL_INNER_SLICE"
-	envVal := "[a, b, c]"
-	os.Setenv(envVar, envVal)
+	envVar := testEnvPrefix + "_INNER_SLICE"
+	os.Setenv(envVar, "[a, b, c]")
 	defer os.Unsetenv(envVar)
-	config := New()
-	config.SetEnvPrefix(Prefix)
 
 	data := "---\nInner:\n    Slice: [d,e,f]"
 
-	err := config.ReadConfig(bytes.NewReader([]byte(data)))
-
-	if err != nil {
-		t.Fatalf("Error reading %s plugin config: %s", Prefix, err)
-	}
+	config := New()
+	config.SetConfigName(testConfigName)
+	err := config.ReadConfig(strings.NewReader(data))
+	require.NoError(t, err, "error reading %s plugin config", testConfigName)
```

Comment:
more out of curiosity as I know this was already there, but why the use of the word `plugin` here?

Reply:
Honestly, I've no idea. When I visit this stuff again (next weekend?) I'll probably modify this as well.